src/lib/dexrt.cpp

```diff
@@ -138,15 +140,49 @@ double randunif(uint64_t keypair) {
   return out - 1;
 }
 
-void showFloat(char **resultPtr, float x) {
-  auto p = reinterpret_cast<char*>(malloc_dex(100));
-  auto n = sprintf(p, "%.4f", x);
-  auto result1Ptr = reinterpret_cast<int32_t*>(resultPtr[0]);
-  auto result2Ptr = reinterpret_cast<char**>(  resultPtr[1]);
-  *result1Ptr = n;
-  *result2Ptr = p;
+// The string buffer size used for converting integer and floating-point types.
+#define STRING_BUFFER_SIZE 32
```

Comment:
nit: this is C++, so you can define `constexpr` globals

Reply:
Thanks, done!